cime_config/machines/Depends.pm-gpu.nvidiagpu.cmake

```diff
@@ -2,9 +2,13 @@ list(APPEND REDUCE_OPT_LIST
   homme/src/share/derivative_mod_base.F90
 )
 
+# add accelerator/gpu flags for MPAS files
```

Comment:
According to @ndkeen's comment, this setting might be placed in the Depends file, not in this cmake file, in order to limit the scope of impact to specified MPAS files.